protocol/contracts/lease/src/position/spec/mod.rs

```diff
@@ -37,42 +37,39 @@ where
         self.liability
     }
 
-    pub fn check_transaction_amount<TransactionC>(
+    /// Check if the amount can be used to close the position.
+    /// Return `error::ContractError::PositionCloseAmountTooSmall` when a partial close is requested
+    /// with amount less than the minimum sell asset position parameter sent on lease open. Refer to
+    /// `NewLeaseForm::position_spec`.
+    ///
+    /// Return `error::ContractError::PositionCloseAmountTooBig` when a partial close is requested
+    /// with amount that would decrease a position less than the minimum asset parameter sent on
+    /// lease open. Refer to `NewLeaseForm::position_spec`.
+    pub fn validate_close_amount<Asset>(
```

Comment:
let's order the public functions as per their usage in the lifecycle of a lease, start with `calc_borrow_amount`, then `check_liability`, and then `validate_close_amount`